libs/ledger-live-common/src/families/casper/bridge/utils/txn.ts

```diff
@@ -37,7 +44,9 @@ export function mapTxToOps(accountId: string, addressHash: string) {
         senders: [fromAccount],
         recipients: [toAccount],
         date,
-        extra: {},
+        extra: {
+          "Transfer ID": transferId,
```

Comment:
I am not really convinced with this. Why are we using object keys with spaces in between?

Reply:
When I used this, the field automatically got displayed in the Operation details. So if it's "transferID" it doesn't look nice.

Reply:
What is mean is, when a key-value is added to extra I think it automatically appear on the operationInformation. Therefore the keys need to be named the way we want them to be displayed in the UI

Reply:
Yes, I remember from other integrations now that you mentioned it.

Reply:
Great!